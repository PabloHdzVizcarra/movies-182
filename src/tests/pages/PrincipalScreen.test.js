import React, { useCallback } from 'react';
import '@testing-library/jest-dom';
import { mount } from 'enzyme';


describe('Test in <PrincipalScreen />', () => {

  const getPopularMovies = async () => {

    try {
      const apiKey = process.env.REACT_APP_APIKEY;

      console.log("OBTENIENDO PELICULAS POPULARES");
      const data = await fetch(
        `https://api.themoviedb.org/3/movie/popular?api_key=${apiKey}&language=es-MX&page=1`
      );
      const { results } = await data.json();

      return results;

    } catch (error) {
      console.log(error);
    }
  };
  
  test('should function call API', async() => {
    
    console.log(await getPopularMovies());

  })
  
})
