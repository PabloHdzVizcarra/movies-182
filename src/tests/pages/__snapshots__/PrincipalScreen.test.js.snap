// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <PrincipalScreen /> the API call must return a movie array  1`] = `
<div
  className="sc-AxhUy ikkLlw animate__animated animate__fadeIn"
/>
`;
